### ISSUE TITLE
[Mask RCNN Swin-S] the loss exploded during training on my dataset

### ISSUE THREAD

**Issue by josef5838** [open-mmlab/mmdetection#11901]
Hi team, thanks for your great work!

I am trying to test if Mask RCNN Swin-S is going to be better than the traditional mask rcnn.

I trained mask rcnn on my dataset with no problem, but when training Mask RCNN Swin-S the gradient exploded after the first 2 epochs and gradient clipping did not help.

I have trained a few times, it always explodes at around Epoch(train)  [2][220/344] despite the seed being different every time.

[20240808_110625.log](https://github.com/user-attachments/files/16541735/20240808_110625.log)
What in your opinion, could be the source of the problem?

Another issue: classic Mask RCNN did not perform super well on my task to extract 1D signals from 2D images, I suspect it is due to the fact that the FCN mask head is not good at classifying objects that have very little area. Do you have suggestions on some new instance segmentation architecture/mask head I should try out? Here's a result from traditional maskrcnn:
![output](https://github.com/user-attachments/assets/78ee6677-6353-421b-90c0-8010da6a9591)

Your help is very much appreciated!
Best
